### PR TITLE
Navigation feature complete

### PR DESCRIPTION
**Pull request by MAstrauskas**
I have finished implementing sticky navigation as per design. None of the links are supposed to work at the moment except the Mindera Gaming logo which should take you to the top of the page once clicked.

### CODE REVIEW

**Comment by pawelgrzybek** [comment]
Overall comments.

Can you put all the source files and directories (scss, js, index.html etc) to `src` folder please?

**Comment by pawelgrzybek** [comment]
Change the order of scss imports please. I will come over and explain.

**Comment by pawelgrzybek** [comment]
If you are not using dynamic characteristic of CSS custom properties, I would convert css variables to Scss, because sass in on the project anyway.

File: index.html [hunk 1]
```diff
@@ -1,13 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
```
Comment (danjordan) [inline index.html]: @pawelgrzybek Do we still need this? I noticed html boilerplate removed it from their index.html

File: index.html [hunk 2]
```diff
@@ -1,13 +1,65 @@
...
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <!-- Navigation bar -->
+    <header>
+      <div class="row row-align-center row-nav-height">
+        <!-- Logo -->
+        <div class="nav-left">
+          <a href="#"
```
Comment (danjordan) [inline index.html]: Are we using prettier? It should fix this going over two lines.

Comment (MAstrauskas) [reply index.html]: I do use prettier. I will fix these issues.

**Comment by MAstrauskas** [comment]
@pawelgrzybek @danjordan I have fixed the issues as per comments and made the navigation accessible to screen readers.

Commit: [db82da3](https://github.com/Mindera/mindera-gaming-website/commit/db82da3461a22bf75246f4e7f454debe387c9b7b)

I would appreciate if you could check it again. Thanks, guys.

**Pull request merged**